mwclient/client.py

```diff
@@ -612,6 +615,156 @@ def get_token(self, type, force=False, title=None):
 
         return self.tokens[type]
 
+    def create_user(self, username, password, **kwargs):
+        if 'createtoken' not in kwargs:
+            kwargs['createtoken'] = self.get_token('createaccount')
+        if 'retype' not in kwargs:
+            kwargs['retype'] = password
+        if 'continue' not in kwargs and 'createreturnurl' not in kwargs:
+            # FIXME should be great if API didn't require this...
```

Comment:
The comment is OK, but please remove "FIXME" if it cannot be fixed without a change in the API.

Reply:
done